### PR TITLE
Create an Escrow in Sway

### LINKED ISSUE #1446: Add an example of an Escrow contract to examples

### PR DESCRIPTION
**Pull request by Braqzen**

### CODE REVIEW

File: examples/escrow/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[project]
+authors = ["Braqzen"]
```
Comment (adlerjohn) [inline examples/escrow/Cargo.toml:2]: For both Cargo and Forc manifest files.
```suggestion
authors = ["Fuel Labs <contact@fuel.sh>"]
```

File: examples/escrow/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+edition = "2021"
+license = "Apache-2.0"
+name = "escrow"
+version = "0.1.0"
```
Comment (adlerjohn) [inline examples/escrow/Cargo.toml:6]: Order based on https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/cargo.md

> Sort key names alphabetically within each section, with the exception of the [package] section. Put the [package] section at the top of the file; put the name and version keys in that order at the top of that section, followed by the remaining keys other than description in alphabetical order, followed by the description at the end of that section.

File: examples/escrow/Forc.toml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[project]
+authors = ["Braqzen"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "escrow"
+
+[dependencies]
```
Comment (adlerjohn) [inline examples/escrow/Forc.toml:7]: For your Forc manifest files, add a local `std` dependency. That will ensure that it's actually using the local branch and therefore actually tests the project against the changeset.

You might need to regenerate the lockfiles.

**Comment by Braqzen** [comment]
Closing this issue as the work is being moved into [Sway Applications](https://github.com/FuelLabs/sway-applications)

**Pull request closed without merging**